File: samples/CS/10-CheckBox/RibbonItems.cs
using System;
using System.Windows.Forms;

namespace WinForms.Ribbon
{
    partial class RibbonItems
    {
        public void Init()
        {
            Button.Click += new EventHandler<EventArgs>(_button_ExecuteEvent);
        }

        void _button_ExecuteEvent(object sender, EventArgs e)
        {
            MessageBox.Show("RibbonCheckBox check status is: " + CheckBox.BooleanValue.ToString());
        }

        public void Load()
        {
        }

    }
}

File: samples/VB/10-CheckBox/RibbonItems.vb
Imports System

Namespace WinForms.Ribbon

    Partial Class RibbonItems

        Public Sub Init()
            AddHandler _Button.Click, AddressOf _button_ExecuteEvent
        End Sub

        Private Sub _button_ExecuteEvent(ByVal sender As Object, ByVal e As EventArgs)
            MessageBox.Show("checkbox check status is: " & _CheckBox.BooleanValue.ToString())
        End Sub

    End Class
End Namespace
